__tests__/app.test.js

```diff
@@ -0,0 +1,35 @@
+const db = require("../db/connection");
+const request = require("supertest");
+const app = require("../app");
+const testData = require("../db/data/test-data");
+const seed = require("../db/seeds/seed");
+
+afterAll(() => {
+  return db.end();
+});
+
+beforeEach(() => {
+  return seed(testData);
+});
+
+describe("far-han-news tests", ()=> {
+    describe("/api/topics", () => {
+        test("GET 200: returns with a status of 200", () => {
```

Comment:
Your test below also expects a 200 and your description also says this so I'm not really sure this is needed as a standalone test.